### PR TITLE
Documentation enhance for Calibration algorithm

### PR DESCRIPTION
**Pull request by yuwenzho**
## Type of Change

documentation
API not changed

## Description

Documentation enhance for Calibration algorithm

## Expected Behavior & Potential Risk

calibration.md should be easy to read

## How has this PR been tested?

CI test

## Dependency Change?

no

### CODE REVIEW

File: docs/source/calibration.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## Introduction
+
+Quantization proves beneficial in terms of reducing the memory and computational requirements of the model. Uniform quantization transforms the input value $x ∈ [β, α]$ to lie within $[−2^{b−1}, 2^{b−1} − 1]$, where $[β, α]$ is the range of real values chosen for quantization and $b$ is the bit-width of the signed integer representation. Calibration is the process of determining the $α$ and $β$ for model weights and activations. Refer to this [link](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization.md#quantization-fundamentals) for more quantization fundamentals
+
+## Calibration Algorithm
+
+There are three main methods of Calibration:
+
+- MinMax: This is the simplest method to get the maximum and minimum of input values as $α$ and $β$ [^1]. This approach maintains the full range but often at the cost of compromising precision.
```
Comment (mengniwang95) [inline docs/source/calibration.md:15]: in my opinion, minmax is also effective and not  'at the cost of compromising precision'

File: docs/source/calibration.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    <td>OnnxRuntime</td>
+    <td>minmax</td>
+    <td>minmax, kl, percentile</td>
+  </tr>
+</tbody>
+</table>
+
+> `kl` is used to represent the Entropy calibration algorithm in Intel® Neural Compressor.
+
+Calibration algorithm is in the tuning space of Intel® Neural Compressor. After tuning, a quantization model with the most suitable calibration algorithm will be returned. Please refer to [tuning_strategies.md](https://github.com/intel/neural-compressor/blob/master/docs/source/tuning_strategies.md#tuning-space) for more details.
```
Comment (yiliu30) [inline docs/source/calibration.md:60]:
```suggestion
The calibration algorithm is one of the tuning items utilized by Intel® Neural Compressor auto-tuning. The accuracy-aware tuning process will select an appropriate algorithm. Please refer to [tuning_strategies.md](https://github.com/intel/neural-compressor/blob/master/docs/source/tuning_strategies.md#tuning-space) for more details.

```

File: docs/source/calibration.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+2. [Calibration Algorithm](#calibration-algorithm)
+3. [Supported Framework Matrix](#supported-framework-matrix)
+
+## Introduction
+
+Quantization proves beneficial in terms of reducing the memory and computational requirements of the model. Uniform quantization transforms the input value $x ∈ [β, α]$ to lie within $[−2^{b−1}, 2^{b−1} − 1]$, where $[β, α]$ is the range of real values chosen for quantization and $b$ is the bit-width of the signed integer representation. Calibration is the process of determining the $α$ and $β$ for model weights and activations. Refer to this [link](https://github.com/intel/neural-compressor/blob/master/docs/source/quantization.md#quantization-fundamentals) for more quantization fundamentals
+
+## Calibration Algorithm
+
+There are three main methods of Calibration:
```
Comment (yiliu30) [inline docs/source/calibration.md:13]: How about changing to `Currently, INC supports three popular calibration algorithms:`? as there are others algo, such as MovingAverage?

**Review by yiliu30 — APPROVED** [review]

**Review by mengniwang95 — APPROVED** [review]

**Comment by chensuyue** [comment]
We need to add the entry point somewhere guide user to this docs. Maybe some where in https://github.com/intel/neural-compressor/blob/master/docs/source/quantization.md, or somewhere directly here https://intel.github.io/neural-compressor/latest/docs/source/user_guide.html#advanced-topics?

File: docs/source/calibration.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+# Calibration Algorithms in Quantization
+
+1. [Introduction](#introduction)
+2. [Calibration Algorithm](#calibration-algorithm)
```
Comment (yiliu30) [inline docs/source/calibration.md:4]:
```suggestion
2. [Calibration Algorithms](#calibration-algorithms)
```

File: docs/source/calibration.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
+# Calibration Algorithms in Quantization
+
+1. [Introduction](#introduction)
+2. [Calibration Algorithm](#calibration-algorithm)
+3. [Supported Framework Matrix](#supported-framework-matrix)
```
Comment (yiliu30) [inline docs/source/calibration.md:5]:
```suggestion
3. [Support Matrix](#support-matrix)
```

**Review by yiliu30 — APPROVED** [review]

**Comment by yuwenzho** [comment]
> We need to add the entry point somewhere guide user to this docs. Maybe some where in https://github.com/intel/neural-compressor/blob/master/docs/source/quantization.md, or somewhere directly here https://intel.github.io/neural-compressor/latest/docs/source/user_guide.html#advanced-topics?

Add link to https://intel.github.io/neural-compressor/latest/docs/source/user_guide.html#advanced-topics

**Pull request merged**